x-pack/plugins/endpoint/server/services/endpoint/endpoint_query_builders.ts

```diff
@@ -65,3 +65,27 @@ async function getPagingProperties(
     pageIndex: pagingProperties.page_index || config.endpointResultListDefaultFirstPageIndex,
   };
 }
+
+export const kibanaRequestToEndpointFetchQuery = (
+  request: KibanaRequest<any, any, any>,
+  endpointAppContext: EndpointAppContext
+) => {
+  return {
+    body: {
```

Comment:
Will this get the latest document for the given Endpoint ID?
The List is using something like the following to ensure only latest is returned:
```javascript
      aggs: {
        total: {
          cardinality: {
            field: 'host.id.keyword',
          },
        },
      },
```

Reply:
it should be sorting by creation, and then grabbing first result. So to your question, yeah, should grab the latest.

That being said, I'm now noticing this is out of date with the endpoint schema anyway

Reply:
Yes, you have to grab the latest. Thanks.

Reply:
updated this to latest endpoint schema, and verified that it grabs just the latest single result, without needing to aggregate (since it's just selecting 1 endpoint, a sort-and-get-1 works fine).

Reply:
Thank you both.

Reply:
ahh - yes, you are right. The aggs was needed because we were retrieving a list. 👍